ArduSub/radio.cpp

```diff
@@ -0,0 +1,152 @@
+// -*- tab-width: 4; Mode: C++; c-basic-offset: 4; indent-tabs-mode: nil -*-
+
+#include "Sub.h"
+
+
+// Function that will read the radio data, limit servos and trigger a failsafe
+// ----------------------------------------------------------------------------
+
+void Sub::default_dead_zones()
+{
+    channel_roll->set_default_dead_zone(30);
+    channel_pitch->set_default_dead_zone(30);
+    channel_throttle->set_default_dead_zone(30);
+    channel_yaw->set_default_dead_zone(40);
+    channel_forward->set_default_dead_zone(30);
+    channel_lateral->set_default_dead_zone(30);
+}
+
+void Sub::init_rc_in()
+{
+	channel_pitch    = RC_Channels::rc_channel(0);
+    channel_roll     = RC_Channels::rc_channel(1);
+    channel_throttle = RC_Channels::rc_channel(2);
+    channel_yaw      = RC_Channels::rc_channel(3);
+    channel_forward  = RC_Channels::rc_channel(5);
+    channel_lateral  = RC_Channels::rc_channel(6);
```

Comment:
I don't think that it was people changing it, so much that we just wanted to narrow down the number of possible causes of issues.